Relativity.DataExchange.Client.SDK.md

```diff
@@ -4,6 +4,17 @@
 
 The Relativity data exchange client SDK .NET package.
 
+## v1.13.18
+
+### Release Notes
+
```

Comment:
Any release notes to share?

Reply:
We checked that only change since 1.13.16 was for RDC only. I'll close this PR